### PR TITLE
[12.0][FIX] Remove unnecessary utf8 coding comments and replace http license links

### PR DESCRIPTION
**Pull request by skeller1**
**Remove linting error `C8202(unnecessary-utf8-coding-comment)` for 12.0 pipeline**

utf8_comments

**Replace old http license links with https**

### CODE REVIEW

File: crm_location_nuts/models/__init__.py [hunk 1]
```diff
@@ -1,3 +1,2 @@
-# -*- coding: utf-8 -*-
```
Comment (NL66278) [inline crm_location_nuts/models/__init__.py:2]: You should remove the blank lines remaining at the beginning of this file (and some others) as well (or replace them with a license line - my personal preference).

File: crm_location_nuts/tests/test_crm_location_nuts.py [hunk 2]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 # Copyright 2017 Tecnativa - David Vidal
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
```
Comment (NL66278) [inline crm_location_nuts/tests/test_crm_location_nuts.py]: And when you are at it anyway: change the license lines to https:

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: Thanks for taking up this legacy stuff. With a few extra changes it will be perfect.

File: crm_location_nuts/tests/test_crm_location_nuts.py (hunk 2 above)
Comment (skeller1) [reply crm_location_nuts/tests/test_crm_location_nuts.py]: Did it, thank you

**Review by skeller1 — COMMENTED** [review]
Ready to go

**Review by skeller1 — COMMENTED** [review]
Changed all license links and removed utf-8 encoding comments from 12.0

**Review by rousseldenis — APPROVED** [review]

File: crm_lead_code/__manifest__.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 #    GNU Affero General Public License for more details.
```
Comment (NL66278) [inline crm_lead_code/__manifest__.py]: For this file replace the whole comment block at the beginning with the shortened license and copyright lines.

File: crm_lead_firstname/__manifest__.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
Comment (NL66278) [inline crm_lead_firstname/__manifest__.py]: And replace Copyright sign with the word `Copyright`.

File: crm_lead_firstname/models/crm_lead.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
Comment (NL66278) [inline crm_lead_firstname/models/crm_lead.py]: Replace copyright sign with word.

File: crm_lead_firstname/tests/test_crm_lead.py [hunk 6]
```diff
@@ -1,5 +1,5 @@
 # © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```
Comment (NL66278) [inline crm_lead_firstname/tests/test_crm_lead.py]: And here and everywhere: Replace copyright sign with word.

File: crm_lead_vat/tests/__init__.py [hunk 7]
```diff
@@ -1,4 +1,4 @@
 # Copyright 2015 Antiun Ingeniería, S.L.
```
Comment (NL66278) [inline crm_lead_vat/tests/__init__.py]: There should be no copyright lines in __init__.py files at all.

**Review by NL66278 — CHANGES_REQUESTED** [review]
:+1: A great cleanup! But now that more files are changed, there are a small number of additional things todo.

**Comment by skeller1** [comment]
@NL66278:

- [x] Removed all copyright comments from init files
- [x] Cleaned up copyright sign

Question:

1. Could I change that line:
`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).`
to
`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).`

2. Could I change: `# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
` to `# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).`

**Comment by NL66278** [comment]
@skeller1 I think all those three lines are valid (and equivalent). I mostly use the agpl.html (without -3.0 but with html), but the shortest version would maybe even be better.

**Review by NL66278 — APPROVED** [review]
:+1: Thank you for all the work!

**Pull request merged**